.github/workflows/test.yml

```diff
@@ -20,8 +25,15 @@ jobs:
         uses: supercharge/mongodb-github-action@1.7.0
         with:
           mongodb-version: ${{ matrix.mongodb-version }}
+      # add caching to reduce bandwidth and save time
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-${{ matrix.python-version}}-pip-${{ hashFiles('**/setup.py') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ matrix.python-version}}-pip-
```

Comment:
Is this needed? How much time are we saving? IIRC these tests run very fast already.

Reply:
The tests do already run fast. I mainly added this in because Steve suggested it and I thought it would be an easy way to make it run a bit faster.

Reply:
Okay and what is the impact of this change?

Reply:
It is a general thing that we add to the Jupyter repos to save time and bandwidth to pypi.org.  If we don't want it here that's fine too.

Reply:
Yeah I get that, I just want to know how much time was actully saved in our case. How slow were the old tests and how much quicker are they now?

Reply:
In this case I see about 40s -> 30s on the Python 3.8 builds.

Reply:
Thanks SGTM!